capabilities/writetarget/report/mercury/v3/types.go

```diff
@@ -0,0 +1,57 @@
+// TODO: duplicate of https://github.com/smartcontractkit/chainlink/blob/develop/core/services/relay/evm/mercury/v3/types/types.go
```

Comment:
Notice that these types are duplicated from core repository to avoid importing core. @silaslenihan could you please scope out a ticket to dedupe this in a follow up. Thx!

cc @jmank88

Reply:
https://smartcontract-it.atlassian.net/browse/BCFR-1226